packages/apolloKeyv/README.md

```diff
@@ -0,0 +1,23 @@
+# `ApolloKeyv` class
+
+The `ApolloKeyv` class is a simple wrapper for `Keyv` which implements the `KeyValueCache` interface from the `@apollo/utils.keyvaluecache` package. This class is intended for use by (but not limited to) Apollo Server's internal cache. `Keyv` supports a number of useful adapters like Redis, Mongo, SQLite, etc.
+
+## Usage
+
+Here's an example of using a Redis store for your ApolloServer cache:
+
+Install the necessary packages:
+
+```bash
+npm install @apollo/utils.apollokeyv keyv @keyv/redis
```

Comment:
is it helpful to have the word apollo twice in this package name? i'm not sure. probably.

Reply:
also perhaps the class should be KeyvKeyValueCache?

Another idea that could help with naming is that, just like with "Fetcher", we could come up with a slightly more "namey" name than KeyValueCache. So then we'd have the Aardvark interface, PrefixingAardvark, InMemoryAardvark, KeyvAardvark, etc. I don't have any particularly good name suggestions though.

Reply:
Dropped `apollo` twice, went with `@apollo/utils.keyvadapter` and `KeyvAdapter` for the class name